apps/betterangels-backend/accounts/admin.py

```diff
@@ -17,6 +21,32 @@
 )
 
 
+class NotesByAdminMixin(object):
+    def notes_by(self, obj: User) -> SafeString:
+        notes = Note.objects.filter(created_by=obj)
+        note_links = [
+            '<a href="{}">{}</a>'.format(
+                reverse("admin:notes_note_change", args=(note.id,)),
+                f"Note {note.id}: {note} (with {note.client.full_name if note.client else None} {note.interacted_at.date()})",
```

Comment:
a lil strange to interpolate with `None`, and also this looks like something that can be put on the Note model so that in can be reused. I see you're interpolating `{note}` directly here, what does that evaluate to? the note title? you can move all of this into a `note_label_with_client` property on note, and then reuse it in other places potentially